src/index.html

```diff
@@ -22,6 +22,624 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <a
+        class="header__menu"
+        href="#menu"
+      >
+        <img
+          src="images/menu.png"
+          alt="menu"
+        />
+      </a>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <aside
```

Comment:
I don't see why you have this aside here. Also, your deployment may be outdated again as it doesn't show in your deployed html:
![image](https://github.com/mate-academy/layout_catalog/assets/52002856/b9fa917a-3467-423d-b327-ed24abfdbb73)

Reply:
I used an aside tag to do a menu when on mobile. I know it's not a requeriment, but I wanted to be completely responsive, if that's okay(?). If not I'll remove it.

Reply:
Also, I talked with Pedro about the deploy and he said it's working fine. Maybe your cache was deleted when testing it?